src/Entity/User.php

```diff
@@ -114,7 +114,7 @@ public function getId(): int
         return $this->id;
     }
 
-    public function setDisplayName(string $displayName): void
+    public function setDisplayName(?string $displayName): void
```

Comment:
Should we call `setDisplayName` with null at all?

Reply:
I had problems with a test where the displayName was " " so just a whitespace.
Due to the trim functionality of Symfony Forms the " " will become a null and that produced a type error because string !== null

Reply:
Ah right! Thanks for explaining this to me 🤗